MapleServer2/Packets/SkillBookTreePacket.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections.Generic;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Types;
+
+namespace MapleServer2.Packets {
+    public static class SkillBookTreePacket {
+        private static int count = 0;
```

Comment:
What is this variable doing?

Reply:
Left over variable from reverse engineering that I forgot to remove